flycheck.el

```diff
@@ -521,20 +521,43 @@ enabled.  Changing it will not affect buffers which already have
   :package-version '(flycheck . "0.15")
   :safe #'booleanp)
 
+(defconst flycheck-minimum-level-choices
```

Comment:
Great suggestion, thanks! I changed to a custom widget. If might be good to add an `:offset` to it, but I'll leave that to your judgement :)

I'm not sure how to make the widget list all currently defined levels though; I don't think we currently keep track of that list. It would be easy to add such bookkeeping to `flycheck-define-error-level`, but it might be slightly confusing if the user selects a level that is defined by an autoloaded package, as the corresponding `severity` will not be known until that package is loaded.